src/univers/version_range.py

```diff
@@ -260,6 +260,37 @@ def from_native(cls, string):
 
         return cls(constraints=anyof_constraints)
 
+    @classmethod
+    def from_gitlab_native(cls, string):
+        constraints = []
+        parsed_constraints = set()
+        pipe_separated_constraints = string.split("||")
+        for pipe_separated_constraints in pipe_separated_constraints:
+            space_seperated_constraints = pipe_separated_constraints.split(" ")
+            constraints.extend(space_seperated_constraints)
+        comparator = ""
+        while "" in constraints:
+            constraints.remove("")
+        for constraint in constraints:
+            if constraint in cls.vers_by_native_comparators:
+                comparator = cls.vers_by_native_comparators[constraint]
+                continue
+            if comparator:
+                parsed_constraints.add(
+                    VersionConstraint(comparator=comparator, version=cls.version_class(constraint))
+                )
+            else:
+                comparator, version_constraint = split_req(
+                    constraint, cls.vers_by_native_comparators
+                )
+                parsed_constraints.add(
+                    VersionConstraint(
+                        comparator=comparator, version=cls.version_class(version_constraint)
```

Comment:
May be a bit more explicit?
```suggestion
                        comparator="=", version=cls.version_class(version_constraint)
```

Reply:
We can not expect the comparator to be "=" every time, it's derived from the string